src/simulation/environment/eclipse/eclipse.cpp

```diff
@@ -283,6 +284,13 @@ double Eclipse::getPlanetEquatorialRadius(std::string planetSpiceName)
         return REQ_URANUS*1000.0;
     } else if (planetSpiceName == "neptune") {
         return REQ_NEPTUNE*1000.0;
+    } else if (planetSpiceName == "custom") {
+        if (REQ_CUSTOM <= 0.0) {
+            bskLogger.bskLog(BSK_ERROR, "Eclipse: Invalid REQ_CUSTOM set.");
+            return 1.0;
+        } else {
+            return REQ_CUSTOM*1000.0;
```

Comment:
`REQ_CUSTOM` should be in units of meters already.  We only used km for the other values because they came from a standard plant into database file.